addon/components/denali-input-group.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="input-group {{this.isStackedClass}} {{this.isResponsiveClass}} {{this.autoWidthClass}}">
+  <label>{{this.label}}</label>
+  {{yield (hash Button=(component "denali-button"))}}
+  {{yield (hash Input=(component "denali-input"))}}
+  {{yield (hash Radio=(component "denali-radio"))}}
+  {{yield (hash Toggle=(component "denali-toggle"))}}
+  {{yield (hash Checkbox=(component "denali-checkbox"))}}
+  {{yield (hash Switch=(component "denali-switch"))}}
+  {{yield (hash Select=(component "denali-select"))}}
```

Comment:
Is there a plan for in the future registering these yielded components so they are controlled somehow by the parent? As it stands considering these input components are toplevel and exposed I don't see a reason to have them be yielded.

If the user is expected to use the top level input components directly there will be no confusion about their API.

Reply:
@jkusa Is there a way to do what Alex is suggesting?  I agree it sounds a little nicer, but I am not aware of how to do this without yields.

Reply:
I believe @Alex-Aralis is suggesting since the input components are top level why yield sub components if you can do:
```hbs
<DenaliInputGroup @label="My Input" >
  <DenaliInput type="text" placeholder="Text Field" />
</DenaliInputGroup>
```

While this is totally valid and possible with or without yielding subcomponents, I feel yielding the sub components is a stronger contract. It says, yes you can put this here and is documented in code. Whether or not the user uses the yielded sub components is up to them.

Reply:
Also, since order does not matter here: we can use a single yield with a single hash